md_api/src/main/java/com/wipro/ats/bdre/md/api/BatchEnqueuer.java

```diff
@@ -32,12 +32,10 @@
 /**
  * Created by SR294224 on 5/29/2015.
  */
-@Component
 public class BatchEnqueuer extends MetadataAPIBase {
     public BatchEnqueuer() {
-        /*ApplicationContext context = new ClassPathXmlApplicationContext("spring-dao.xml");
-        AutowireCapableBeanFactory acbFactory = context.getAutowireCapableBeanFactory();
-        acbFactory.autowireBean(this);*/
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)